### DISCUSSION TITLE
it is raising a error

### DISCUSSION THREAD

**Discussion by balajz** [tecoholic/ner-annotator#128] [Q&A]
AssertionError: [E923] It looks like there is no proper sample data to initialize the Model of component 'tok2vec'. To check your input data paths and annotation, run: python -m spacy debug data config.cfg and include the same config override values you would specify for the 'spacy train' command.

**Comment by tecoholic** [comment]
@Balaji01-4D Can you kindly provide more details? From what you have posted, this seems to be a Spacy command line error and not something that's raised in the NER Annotator.

What is the command that's causing this error? If this is happening when you are using the JSON you obtained from NER Annotator, would it be possible to share the file that's causing the error?

**Reply by balajz** [reply]
first of all thank you for quick response . i used the ner annotator and i exported as .json file and converted to .spacy by command spacy convert train_data.json ./ --ner
 . i tried to train my model and that is the error raised . neega tamil ah bro ?? . could you explain me how to use this if did it in a wrong way becos there is not much  source except chatgpt .

**Reply by tecoholic** [reply]
@Balaji01-4D I see. It looks like you are having trouble using the JSON file. Kindly see the [FAQ section](https://github.com/tecoholic/ner-annotator?tab=readme-ov-file#faq) for guides on how to use the JSON file correctly. Specifically the first one - *How do I import the annotations to spaCy?*
